client/src/Movies/Movie.js

```diff
@@ -1,18 +1,20 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 
+import { useParams } from 'react-router-dom';
+
 export default function Movie(props) {
   const [movie, setMovie] = useState();
+  const { addToSavedList } = props;
 
-  let id = 1;
+  const {id} = useParams();
```

Comment:
You should use Prettier for code formatting, it will automatically add spaces and make the code easier to read